### PR TITLE
Commenting policy

### PR DESCRIPTION
**Pull request by gvwilson**
For @jduckles to review.

### CODE REVIEW

File: pages/commenting.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  and comments however we see fit both for commercial and
+  non-commercial distribution. If this concerns you, use a pseudonym
+  and remove identifying details before you send, or simply post a
+  link to your own blog.  Alterations will rarely be made and will
+  always be obvious; see
+  <a href="http://sarah.thesharps.us/2015/10/05/closing-a-door/#comment-147828">this, for example</a>.
+</p>
+<p>
+  We do not share your personal email address or identity, with one
+  exception: people who threaten or harass me or other posters will
```
Comment (jduckles) [inline pages/commenting.html]: s/me or other posters/community members/

File: pages/commenting.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+<p>
+  We moderate things in these here parts. Not every point of view
+  deserves to be heard. Unmoderated spaces tend to be full of jerks.
+</p>
+<p>
+  Anonymous posting is allowed. Using a consistent identity is encouraged.
+</p>
+<p>
+  Your comments will theoretically show up immediately, though the spam
+  filter may eat legitimate comments, even by regular posters.  Alert use
```
Comment (jduckles) [inline pages/commenting.html]: s/use/us

**Comment by jduckles** [comment]
Just those two fixes and then I think it is ready to go. Thanks!

**Comment by gvwilson** [comment]
@jduckles over to you...

**Comment by jduckles** [comment]
Looks good, builds cleanly on my end.

**Pull request merged**